heron/tools/tracker/src/python/query_operators.py

```diff
@@ -476,7 +477,7 @@ async def execute(self, tracker, tmanager: TManagerLocation, start: int, end: in
       # Initialize with first metrics timeline and its instance
       met = Metrics(None, None, metric.instance, start, end, metric.timeline.copy())
       for timestamp in list(met.timeline.keys()):
-        v = self._f(met.timeline[timestamp], metrics2[""].timeline.get(timestamp))
+        v = self._f(met.timeline[timestamp], metrics2[""].timeline[timestamp])
```

Comment:
Good change. It is better to use the `[]` instead of the `.get()` method throughout to access the dictionary elements for consistency.

Reply:
Reverting. Please see the comment above.